### PR TITLE
candy-icons: unstable-2023-12-31 -> 0-unstable-2024-10-22

### PR DESCRIPTION
**Pull request by arunoruto**
First PR, if I made some mistakes feel free to correct me :)

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [x] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

cc @clr-cera 

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by arunoruto** [comment]
Result of `nixpkgs-review pr 351624` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>candy-icons</li>
  </ul>
</details>

**Comment by arunoruto** [comment]
@tomberek

**Review by zakame — DISMISSED** [review]
Hello, thanks for the PR and welcome to the community! 🎉 

I recall the `maintainer-list.nix` file having its own formatting rules, so please undo the formatting changes, other than the addition of your identity to the list.  Thank you! 🙇

**Comment by arunoruto** [comment]
> Hello, thanks for the PR and welcome to the community! 🎉
> 
> I recall the `maintainer-list.nix` file having its own formatting rules, so please undo the formatting changes, other than the addition of your identity to the list. Thank you! 🙇

Hi, thanks for the hint! I guess it is fixed now!

File: pkgs/by-name/ca/candy-icons/package.nix [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  gtk3,
 }:
 
 stdenvNoCC.mkDerivation {
   pname = "candy-icons";
-  version = "unstable-2023-12-31";
+  version = "2024-10-22";
```
[the remark this answers is not in the document]
Comment (arunoruto) [reply pkgs/by-name/ca/candy-icons/package.nix]: Thanks for the heads up, I wlll note this down for the future!
I corrected the version now :)

**Comment by clr-cera** [comment]
Looks good to me!

**Comment by jtojnar** [comment]
For the record the upstream diff: https://github.com/EliverLara/candy-icons/compare/e4464d7b4d8e1821025447b2064b6a8f5c4c8c89...f8a4125d338d86a58723ab36f9f827248d07c85c

The only non-icon changes are https://github.com/EliverLara/candy-icons/commit/faea9d01b5c46f9dc6c669b4f59e5d83bc09aa97 and https://github.com/EliverLara/candy-icons/commit/fac60f0ccccd4761565345905951df3b66ec29dc to `index.theme`.

**Pull request merged**